source/manual/republishing-content.html.md

```diff
@@ -34,6 +38,19 @@ For example:
 You may wish to test first on Integration:
 [`publishing_api:bulk_republish:document_type[DocumentClass]`][republish-whitehall-document-type-jenkins-integration]
 
+For a short list of Content IDs, run:
+[`publishing_api:bulk_republish:republish_documents_by_content_ids[content_id_1 content_id_2]`][republish-whitehall-document-type-jenkins]
```

Comment:
The URL portion of this needs updating - it currently refers to a different rake task.